config/types/files_test.go

```diff
@@ -0,0 +1,43 @@
+package types
+
+import (
+	"github.com/vincent-petithory/dataurl"
+	"strings"
+	"testing"
```

Comment:
3rd party imports are usually separated from stdlib imports.
```suggestion
	"strings"
	"testing"
	
	"github.com/vincent-petithory/dataurl"
```